math/AggresiveCows.cpp

```diff
@@ -0,0 +1,51 @@
+#include <iostream>
```

Comment:
Please format your code according to this [code style conventions](https://google.github.io/styleguide/cppguide.html)